requests/src/main/scala/weco/api/requests/responses/LookupPendingRequests.scala

```diff
@@ -21,16 +25,18 @@ trait LookupPendingRequests extends CustomDirectives {
         userHolds <- sierraService.getStacksUserHolds(patronNumber)
 
         holdsWithCatalogueIds <- Future.sequence(
-          userHolds.right.get.holds.map { hold =>
+          userHolds.right.get.holds.map { hold: StacksHold =>
             itemLookup
               .bySourceIdentifier(hold.sourceIdentifier)
               .map {
-                case Left(elasticError) =>
-                  warn(s"Unable to look up $hold in Elasticsearch")
+                case Left(elasticError: ElasticsearchError) =>
+                  warn(
+                    s"Unable to look up $hold in Elasticsearch: ${elasticError}"
```

Comment:
Let's not address it here for the sake of getting this PR out quickly, but having a sourceIdentifier on the hold feels a bit odd to me – a hold is a thing in Sierra and could have a source identifier, it's just not the identifier we're using here.

Reply:
What will happen in this branch if we can't find the item?

Reply:
> but having a sourceIdentifier on the hold feels a bit odd to me

I agree, we use it to look up the item later and fill the Item option on `StacksHold` which itself feels wrong. I expect that it'll get removed refactoring out the `.get`.

> What will happen in this branch if we can't find the item?

Good point. I think this is actually a case that will blow up the `.get` on the option added in this PR (the`StacksHold` is returned with a `None` for an `Item`. It's still quite unlikely but we should handle this by doing a multi-item get and returning a list of the items we can get while logging that things are missing with a `warn`.